lib/dockerize_rails/docker_helper.rb

```diff
@@ -5,5 +5,104 @@ module DockerHelper
     def self.version
       Docker.version
     end
+
+    def self.pull
+      status = 0
+      Docker::Image.create('fromImage' => "ruby:#{DRConfig.ruby_version}")
+      puts "Base image 'ruby:#{DRConfig.ruby_version}' ready".green
+      status += pull_mysql
+      status += pull_postgres
+      status
+    rescue Docker::Error::NotFoundError => e
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'e' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->